### ISSUE TITLE
"Check All" button not behaving correctly

### ISSUE THREAD

**Issue by MathewWelsh** [latenitefilms/BRAWToolbox#196]
Two part question here. 
Part #1 
I have succeeded once in setting some parameters (Camera Metadata like Gamut, Colour Space, etc.) and applying (copying) those parameters across all clips listed in the BRAW Toolbox import prep window. I had to mess around with it quite a bit before it worked, but I got it to work. ONCE. But now I cannot replicate the steps to save my life! I forget how I did it. "Check all" "Paste Checked Items" "Copy Checked Items" "Apply Checked to all" --

I've got around in circles trying different combinations of these buttons, and to no avail. I cannot set parameters on one clip, then have the parameters applied across all the clips. Please help. Thanks.

Part #2
In your video https://www.youtube.com/watch?v=cubE8VwZRh4 you show at one point (around 5:30) that your inspector has some BRAW Toolbox controls for colour, etc. But I cannot replicate that in my FCP 11. No matter what I select, clips in browser, on timeline, etc, I do not see that effects/controller box. 
<img width="1415" alt="Screenshot 2024-11-22 at 9 16 09 PM" src="https://github.com/user-attachments/assets/cf9611af-fdc3-4ab7-a365-3b2c16586849">
<img width="1315" alt="Screenshot 2024-11-22 at 9 17 54 PM" src="https://github.com/user-attachments/assets/5e36a92b-f36e-4612-ba21-6c574e4d7743">

**Comment by latenitefilms** [comment]
> **Part 1:**
> I have succeeded once in setting some parameters (Camera Metadata like Gamut, Colour Space, etc.) and applying (copying) those parameters across all clips listed in the BRAW Toolbox import prep window. I had to mess around with it quite a bit before it worked, but I got it to work. ONCE. But now I cannot replicate the steps to save my life! I forget how I did it. "Check all" "Paste Checked Items" "Copy Checked Items" "Apply Checked to all" --

This is explained in more detail here:

https://brawtoolbox.io/workflow-extension/#inspector

The checkboxes on the right are selection checkboxes. Most of the buttons underneath only work in relation to what is checked. For example, it'll only copy the checked items, it'll only paste the checked items.

> **Part 2:**
> In your video https://www.youtube.com/watch?v=cubE8VwZRh4 you show at one point (around 5:30) that your inspector has some BRAW Toolbox controls for colour, etc. But I cannot replicate that in my FCP 11. No matter what I select, clips in browser, on timeline, etc, I do not see that effects/controller box.

If you have a Multicam Clip, double click on it.

Inside the Multicam Clip is a Synchronised Clip. Double click on it. If you don't have **Create Multicam Clips** selected, BRAW Toolbox will create Synchronised Clips.

Inside the Synchronised Clip is a dummy generator on the primary storyline. Click on it. On this generator is the BRAW Toolbox effect. This is where you adjust the RAW controls.

This is explained in more detail here:

https://brawtoolbox.io/how-to-use/

**Comment by MathewWelsh** [comment]
Thanks, The part two of my comment was answered with your reply. Great!

but Part 1 re: applying parameters in the inspector to all clips in the Toolbox browser is not working. 

I made a Vimeo video to show what's going on. https://vimeo.com/1033134451/52089834b0?share=copy

Please have a look and let me know what I'm doing wrong, thanks!

**Comment by MathewWelsh** [comment]
By experimenting further, what I've been able to do is to check boxes manually, say three parameters (like Colour Space/ Gammut, Gamma, gammut suppression) THEN I click "Apply Checked to ALL" and it works. The other clips in the Toolbox Import Browser all have the parameters applied. But when I set parameters, then click "Check All" button to select all parameters, the Apply Checked to ALL button does NOT work. This seems to be a bug in the system.
